src/main/scala/scorex/crypto/authds/avltree/batch/serialization/ProxyInternalNode.scala

```diff
@@ -1,9 +1,13 @@
 package scorex.crypto.authds.avltree.batch.serialization
 
+import scorex.crypto.authds.avltree.batch.InternalNode.InternalNodePrefix
 import scorex.crypto.authds.avltree.batch.{InternalProverNode, ProverNodes}
 import scorex.crypto.authds.{ADKey, Balance}
 import scorex.crypto.hash.{CryptographicHash, Digest}
 
+/**
+  * Internal node for which not children are stored but just their digests
+  */
 class ProxyInternalNode[D <: Digest](protected var pk: ADKey,
                                      val selfLabelOpt: Option[D],
```

Comment:
I wonder why do we need selfLabelOpt at all in ProxyInternalNode?
Why can't we use labelOpt from base class?
Before this PR the difference was that `ProxyInternalNode.label` method always call computeLabel if label was not defined (although  it seems to be more likely bug rather than feature). Now the logic of label evaluation is exactly the same as in Node trait.

Reply:
Good catch, not sure what was motivation for selfLabelOpt, but it seems now it is not needed. Removed.